src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2835,6 +2838,11 @@ void processContextElement
       responseP->contextElementResponseVector.push_back(cerP);
     }
   }
+  else if (apiVersion == "v2" && action == "APPEND" && !isUpdate)
```

Comment:
I see in src/lib/serviceRoutinesV2/ two differents files for this operation. First postEntities.cpp and with some debug I saw that this file is used, the other file is postEntity.cpp but when this file is used? if you see the name seems to be for same operation but one for many entities and other for a single entity. BTW postEntity.cpp use APPEND_STRICT